index.html

```diff
@@ -1,53 +0,0 @@
-<html>
```

Comment:
index.html is still required, but it must contain only a redirect. Here's an example: https://github.com/apolukhin/type_index/blob/develop/index.html